src/components/RolesAndSkills.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react';
+import { Input, Radio, Form, Select, Option, InputNumber, Button } from 'antd';
+
+/* eslint-disable no-template-curly-in-string */
+
+const validateMessages = {
+  required: '${label} is required!',
+};
+
+/* eslint-disable no-template-curly-in-string */
+
+const RolesAndSkills = () => {
+  const [componentSize, setComponentSize] = React.useState('default');
+
+  const onFormLayoutChange = ({ size }) => {
+    setComponentSize(size);
+//
+  const[form] = Form.useForm();
+  const [requiredMark, setRequiredMarkType] = useState('optional');
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'requiredMark' is assigned a value but never used.